text/0000-prepare-phase.md

```diff
@@ -0,0 +1,105 @@
+# Meta
+[meta]: #meta
+- Name: Prepare Phase
+- Start Date: 2021-01-13
+- Author(s): [@jkutner](github.com/jkutner/)
```

Comment:
Should i write my name as author ?

Reply:
or will it stay jkutner, because its owner of main issue.

Reply:
Not sure, maybe put all of us as co-authors for now & we could confirm if that's the right thing.